setup.py

```diff
@@ -95,6 +95,8 @@ def get_hierarchy():
 
 try:
     LIKWID_PREFIX, LIKWID_LIBPATH, LIKWID_LIB, LIKWID_INCPATH = get_hierarchy()
+    # TODO
+    major, minor, release = (5, 3, 2)
```

Comment:
Hardcoded version is not future-proof. The major and minor number is present in `likwid.h` after installation, so maybe we read it from there.
Remark: There is no version 5.3.2